tests/commands/help.spec.ts

```diff
@@ -59,9 +59,9 @@ Additional commands:
     device shutdown <uuid>                shutdown a device
     devices supported                     list all supported devices
     env add name [value]                  add an environment or config variable to an application or device
-    env rename <id> <value>               rename an environment variable
+    env rename id value                   change an environment variable for an application or device
```

Comment:
"rename" is a bit misleading, because the command changes the variable's **value** rather than its **name.** Perhaps the verb should have been "set".

Reply:
🤦‍♂ This is the kind of mistake that will haunt us for a long time - is there a list of breaking changes we want to make anywhere? We should definitely make a not of this

Reply:
I've now created a GDoc to keep track of such suggested changes: https://docs.google.com/document/d/1uHE-_LLu5gJUQZICQJtlo76EQCm90z0CMAwpkfw6s8w/edit?usp=sharing
I've also further improved the command description to include _"change the value of..."_ so that at least the description is clear about what the command does.